magefile.go

```diff
@@ -114,6 +113,25 @@ var (
 	goIntegTestTimeout = 2 * time.Hour
 	// goProvisionAndTestTimeout is the timeout used for both provisioning and running tests.
 	goProvisionAndTestTimeout = goIntegTestTimeout + 30*time.Minute
+
+	helmChartsValues = []struct {
+		path        string
+		versionKeys []string
+		tagKeys     []string
+	}{
+		// elastic-agent Helm Chart
+		{
+			"./deploy/helm/elastic-agent",
+			[]string{"agent", "version"},
+			[]string{"agent", "image", "tag"},
+		},
+		// edot-collector values file for kube-stack Helm Chart
+		{
+			"./deploy/helm/edot-collector/kube-stack",
```

Comment:
nit: maybe have the path as a const?!

Reply:
https://github.com/elastic/elastic-agent/pull/5822/commits/16fdafc196f65b36864840084db15499b0e99ed3